### ISSUE TITLE
Extend search in automatic suggestion tab from translation memory only to any machinery

### ISSUE THREAD

**Issue by rhofer** [WeblateOrg/weblate#7345]
## Situation

During translation, the "Automatic suggestions" tab provides powerful tooling and according machinery results, which are giving a major contribution to translators efficiency. It provides a search field in order to play with deviations / alternatives to the given source string.

## Goal

As a translator, I want to use the search field in the "Automatic suggestions" tab in order to benefit from the "Copy" or "Copy and save" actions based on the provided results. Therefore, I expect to receive search results from any configured machinery, not limited to translation memory.

## Problem

The search field only provides results given by `Weblate Translation Memory` machinery. This might be a relict from the times where we had a `Machine translation` tab and a `Translation memory` tab which were merged to `Automatic suggestion` tab.

## Solution Approach

Extend this search to include any configured machinery.

Optional:
- offer options to search in TM only or to include other machineries
- could be realized with checkbox, or, two buttons `search all` and `search TM`
- Question is, how fine granular the selection of option shall be offered. I.e. shall it be possible to select/deselect any machinery specifically? > this can result in many checkboxes.

**Comment by nijel** [comment]
This is not relict, it's caused by the behavior - you can not "search" in the machine translation, you can search only in memory like services.

**Comment by rhofer** [comment]
But when you click on "Automatic suggestion" tab, then results are received from any machinery. Why can the same not be provoked when entering a search string in the local search field belonging to the "Automatic suggestion" tab?

**Comment by nijel** [comment]
Yes, but the search can be for any term, it doesn't have to be term present in the current results.

With memory it is possible to search for anything and return matching phrases.

With machine translation, we prefer not to offer free form interface for translating - it usually has attached costs.

**Comment by rhofer** [comment]
@nijel thanks for your clarification. Would it then be possible to include `weblate translation memory` and `weblate` machinery in addition, but still skipping external ones as deepL?

**Comment by nijel** [comment]
Yes, that could be the way to go.
